distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -251,7 +251,9 @@ public void testMetacardTransformersFromRest() {
                 .log()
                 .all()
                 .assertThat()
-                .body(hasXPath("/metacard[@id='" + id + "']"));
+                .body(hasXPath("/metacard[@id='" + id + "']"))
+                .body(hasXPath("/metacard/string[@name='point-of-contact']"),
+                        containsString("Guest@Guest"));
```

Comment:
I'm not sure exactly what that is, but I used this one because I can only check that the xml contains a string (rather than exact string equality) because the `point of contact` includes other information specific to the host. I didn't think it'd be a good idea to depend on the host information.